Gaijinizer patch/Enemies.txt

```diff
@@ -2460,10 +2461,12 @@ number of tails were sit tass.#MTLed
 同種ではないが、中毒性の高い蜜を分泌する。
 ツタでの拘束や形状に至るまで、ランパスを模してある
 > CONTEXT: Enemies/600/BookDescription/4
-One of Dakchi's makeover forms.
-It is possible to generate pollen similar to Lampas.
-It is not homologous but secretes highly addictive honey.
-The lampas are modeled after the bound and shape of the ivy#MTLed
+One of Daji's many forms. Like Lampas, she can
+release pollen into the air. She also secretes a
+nectar that unlike Lampas', is highly addictive.
+What does strongly resemble Lampas however, is
+her overall appearance and the ivy she uses to
+bind her opponent.#RonIffyWording
 > END STRING
```

Comment:
A bit of flow here:

>_**One of Daji's many forms. Like Lampas, she can
release pollen into the air. She also secretes a
nectar that unlike Lampas's, is highly addictive.
What more strongly resembles Lampas however, is
her overall appearance and the ivy she uses to
bind her opponent.**_